client/state/initial-state.js

```diff
@@ -48,7 +48,24 @@ function deserialize( state ) {
 	return reducer( state, { type: DESERIALIZE } );
 }
 
-function loadInitialState( initialState ) {
+function addSympathy( initialStateLoader ) {
+	if ( 'development' !== process.env.NODE_ENV || ( Math.random() > 0.5 ) ) {
+		return initialStateLoader;
+	}
+
+	console.log( 'Skipping initial state load to recreate first-load experience.' ); // eslint-disable-line no-console
```

Comment:
looks like the env badge might be set in the server-side which would make it hard to do here I think.

wanted to avoid `debug` since you have to turn that on but console will always be there